### DISCUSSION TITLE
[Feature Request] Add support for Redshift Spectrum tables

### DISCUSSION THREAD

**Discussion by rubenssoto** [apache/superset#19185] [Ideas]
Hello Guys,

I can't see redshift spectrum tables on sql lab:

<img width="405" alt="Captura de Tela 2021-07-17 às 03 05 51" src="https://user-images.githubusercontent.com/36298331/126027643-0dab29c4-6c16-4204-bcad-ed9d41cbcec3.png">

If I choose local schema, it works fine

<img width="375" alt="Captura de Tela 2021-07-17 às 03 06 27" src="https://user-images.githubusercontent.com/36298331/126027647-dfa42636-4953-4b38-99d1-dd7767707c28.png">

Anyone knows how to solve this?

thank you so much!

**Comment by srinify** [comment]
@rubenssoto thanks for submitting! Moving this feature request to Github Discussions so we can keep Issues focused on bugs!

**Comment by cesar-loadsmart** [comment]
Hi Everyone!

Any updated on this? 

I saw that some other tools like TIBCO, need this extra parameter:  `<table-types>TABLE, VIEW, EXTERNAL TABLE</table-type>` 

More details here: https://support.tibco.com/s/article/Amazon-Redshift-Spectrum-tables-are-not-visible-when-connected-from-Information-Designer-in-TIBCO-Spotfire

I also saw something about including this property: `options=--include-external-accounts.`

But none of them worked for me.

**Comment by mdeshmu** [comment]
I think it depends on the driver you are using to connect to redshift. There was a recent support added for spectrum tables in redshift-connector amazon's official driver for redshift but It is not yet released. It is likely to be released in this month. I recommend testing with it once it releases and update here your findings.
